fmrest/server.py

```diff
@@ -627,6 +627,115 @@ def last_script_result(self) -> Dict:
             }
         return result
 
+    def get_product_info(self) -> Dict:
+        """Fetches product info and returns Dict instance
+
+        Parameters
+        -----------
+        none
+        """
+        path = API_PATH['meta']['product']
+
+        # There is never any auth for retrieving the producInfo, 
+        # so let's keep the Auth header 'clean'...
+        #self._headers.pop('Authorization', None)
+
+        # _call_filemaker() will reinstate the Auth header 
+        # as long as _token is set
+        response = self._call_filemaker('GET', path)
+
+        # Reset the Auth header to its previous state
+        # but _call_filemaker does this anyway...
+        #self._update_token_header()
```

Comment:
We could remove the old comments and commented code here as we won't pop the Auth header anymore.